assets/src/scripts/index.spec.js

```diff
@@ -86,3 +86,10 @@ import './web-services/models.spec';
 import './web-services/nldi-data.spec';
 import './web-services/observations-spec';
 import './web-services/statistics-data.spec';
+
+import './networks/web-services/network-data.spec';
```

Comment:
@gpetrochenkov-usgs , I added the tests for the networks directory here. It seemed simpler to do this then have a separate karma.conf.js and also the coverage report was not getting generated correctly. Now we can see the coverage (and it's pretty good!).